app/plugins/user_interface/rest_api/browse.js

```diff
@@ -0,0 +1,361 @@
+'use strict';
+
+var libQ=require('kew');
+var _=require('underscore');
+var moment=require('moment');
+
+module.exports = RESTApiBrowse;
+
+function RESTApiBrowse(context) {
+    var self = this;
+
+
+    // Save a reference to the parent commandRouter
+    self.context=context;
+    self.logger=self.context.logger;
+    self.commandRouter = self.context.coreCommand;
+}
+
+RESTApiBrowse.prototype.browseListing=function(req,res)
+{
+    var path;
+
+    if(req.query && req.query.path)
+        path=decodeURIComponent(req.query.path);
+    else path='/';
+
+    var defer;
+
+    var ipAddress=this.commandRouter.sharedVars.get('ipAddress');
+
+    if(path=='/')
+    {
+        var content={"navigation":{
+            "lists":[{
+                "items":[]
+            }]
+        }};
+
+
+        content.navigation.lists[0].items.push({"title":"MusicLibrary",
```

Comment:
@fanciulli We should not hardcode those, but take them from the getBrowsesources:
https://github.com/volumio/Volumio2/blob/master/app/index.js#L236

This way you'll have them complete & translated